src/algebra/big-integer.md

```diff
@@ -1,16 +1,16 @@
-<!--?title Long Arithmetics -->
+<!--?title Arbitrary-Precision Arithmetic -->
 
-# Long Arithmetics
+# Arbitrary-Precision Arithmetic
 
-Long arithmetics is a set of data structures and algorithms which allows to process much greater numbers than can be fit in standard data types. Here are several types of long arithmetics.
+Arbitrary-Precision arithmetic, also known as "bignum" or simply "long arithmetic" is a set of data structures and algorithms which allows to process much greater numbers than can be fit in standard data types. Here are several types of arbitrary-precision arithmetic.
 
-## Classical Integer Long Arithmetics
+## Classical Integer Long Arithmetic
```

Comment:
Do you want to replace long arithmetic with arbitrary-precision in all headers, for consistency?

Reply:
Honestly, no :) I wish to preserve some of "long arithmetic" and "arbitrary precision arithmetic" snippets together, hoping that it will help search engines to direct people to our site more often. Moreover that it seems the part of community which is not native-English speaking really uses "long arithmetic" term often...

Reply:
I'm not sure we should be encouraging people to use wrong terminology, even for the sake of search engines, but I can live with this.